README.md

```diff
@@ -48,298 +48,400 @@ The FWS Payments contract enables ERC20 token payment flows through "rails" - au
+An operator is a smart contract—typically the main contract for a given service—that manages payment rails on behalf of clients. A client must explicitly approve an operator and grant it specific allowances, which act as a budget for how much the operator can spend or lock up on the client's behalf.
 
-The operator:
+The operator role is powerful, so the operator contract must be trusted by both the client and the service provider. The client trusts it not to abuse its spending allowances, and the service provider trusts it to correctly configure and manage the payment rail.
 
-- Creates rails from clients to service providers.
-- Changes payment rates, lockups, etc. of payment rails created by this operator.
-  - The sum of payment rates across all rails operated by this contract for a specific client must be at most the maximum per-operator spend rate specified by the client.
-  - The sum of the lockup across all rails operated by this contract for a specific client must be at most the maximum per-operator lockup specified by the client.
-- Specifies / changes the payment rail validator of payment rails created by this operator.
-- Terminates payment rails.
+An approved operator can perform the following actions:
 
-### Per-Rail Lockup: Streaming and Fixed Buckets
+-   **Create Rails (`createRail`):** Establish a new payment rail from a client to a service provider, specifying the token, payee, and an optional validator.
+-   **Modify Rail Terms (`modifyRailLockup`, `modifyRailPayment`):** Adjust the payment rate, lockup period, and fixed lockup amount for any rail it manages. Any increase in the client's financial commitment is checked against the operator's allowances.
+-   **Execute One-Time Payments (`modifyRailPayment`):** Execute one-time payments from the rail's fixed lockup.
+-   **Settle Rails (`settleRail`):** Trigger payment settlement for a rail to process due payments. As a rail participant, the operator can initiate settlement at any time.
+-   **Terminate Rails (`terminateRail`):** End a payment rail. Unlike clients, an operator can terminate a rail even if the client's account is not fully funded.
 
-Each payment rail requires the user to manage two lockup components:
+### Per-Rail Lockup: The Guarantee Mechanism
 
-- **Payment Stream lockup:** `paymentRate × lockupPeriod` (covers future payment streams)
-- **Fixed lockup:** `lockupFixed` (covers one-time payments)
+Each payment rail requires the client to lock funds to guarantee future payments. This lockup is composed of two distinct components:
 
-The contract always enforces that the sum of these two values is locked for each rail:
+-   **Streaming Lockup (`paymentRate × lockupPeriod`):** A calculated guarantee for rate based payments for a pre-agreed lockup period.
```

Comment:
I'm not sold on "Streaming Lockup" although I don't have an alternative, it would be good to check this with people less deeply involved in this to see if they have ideas. "Payment Stream Lockup" felt a bit closer, "Streaming" alone has too many connotations unrelated to payments that can be brought to bear mentally when parsing this I worry.

Reply:
@rvagg No strong opinions though I like "Streaming Lockup". But we can keep this open till we have more feedback from the outside world.